src/main/java/org/killbill/billing/plugin/core/config/YAMLPluginTenantConfigurationHandler.java

```diff
@@ -69,11 +69,13 @@ public void setDefaultConfigurable(final T defaultConfigurable) {
     @Override
     protected void configure(@Nullable final UUID kbTenantId) {
         final String rawConfiguration = getTenantConfigurationAsString(kbTenantId);
+        T configurable = null;
```

Comment:
Shouldn't `PluginTenantConfigurableConfigurationHandler` be updated as well (it's the key-value version of the YAML one)?

Reply:
936841a